compiler/Bond/Template/CustomMapping.hs

```diff
@@ -42,9 +50,9 @@ integer = decimal <$> many1 digit <?> "decimal number"
 -- parse alias mapping specification from the command line --using flags
 -- e.g.: --using="OrderedSet=SortedSet<{0}>"
 parseAliasMapping :: [String] -> IO [AliasMapping]
-parseAliasMapping = mapM parseAliasMapping
+parseAliasMapping = mapM parseAliasMapping'
```

Comment:
I really don't like such name shadowing, basically for 2 reasons:
1. it introduces a warning that can refer to a real error
2. it looks a bit complicated and forces user to look at the context as `foo = mapM foo` it looks like some scary thing with infinite type :).

I'd use `go` or worker or just inlined it as:

```haskell
parseAliasMapping = mapM (\s -> either (fail.show) return (parse aliasMapping s s)
```

Not sure what solution will require less brain cycles from the person who reads it..